cli/src/diff_util.rs

```diff
@@ -2047,14 +2055,14 @@ pub fn show_diff_stats(
     };
 
     for (stat, ui_path) in iter::zip(stats.entries(), &ui_paths) {
-        let bar_added = (stat.added as f64 * factor).ceil() as usize;
-        let bar_removed = (stat.removed as f64 * factor).ceil() as usize;
+        let bar_added = (stat.added as f64 * factor).round() as usize;
+        let bar_removed = (stat.removed as f64 * factor).round() as usize;
```

Comment:
Maybe we should use `.floor()`? If both added/removed numbers are rounded up, I think the bar width exceeds the allocated space.